frontend/tailwind.config.ts

```diff
@@ -3,7 +3,9 @@ const { nextui } = require('@nextui-org/react')
 
 const config: Config = {
   content: [
-    './src/**/*.{js,ts,jsx,tsx,mdx}',
+    './pages/**/*.{js,ts,jsx,tsx,mdx}',
+    './components/**/*.{js,ts,jsx,tsx,mdx}',
+    './layouts/**/*.{js,ts,jsx,tsx,mdx}',
```

Comment:
directory for all the JSX files. Turns out, ./src doesn't work thus, tailwind cannot find our frontend files